File: 240127/2개 이상의 알파벳/more-than-one-alphabet.java
import java.io.*;

import java.util.*;


public class Main {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
    static StringTokenizer st;
    static StringBuilder sb = new StringBuilder();

    public static void main(String[] args) throws IOException {


        String str = br.readLine();

        valid(str);


        bw.write(sb.toString());
        bw.flush();
        bw.close();

    }

    private static void valid(String str){
        int cnt = 0;
        boolean flags = false;

        for(int i = 0; i < str.length(); i++){
            for(int j = 0; j < str.length(); j++){
                if(i != j){
                    if(!(str.substring(i,i+1).equals(str.substring(j,j+1)))){
                        cnt++;
                    }
                }
            }
            if(cnt >= 2){
                flags = true;
                break;
            }
        }

        if(flags){
            sb.append("Yes");
        }else{
            sb.append("No");
        }

    }




}

File: 250615/2개 이상의 알파벳/more-than-one-alphabet.py
A = input()

def valid (a) :
    
    cnt = 0;
    for i in range(0, len(a)) :
        for j in range(1, len(a)) :
            if a[i] != a[j] :
                cnt += 1;
                break;
        if cnt >= 2 :
            return True;
    
    return False;

if valid(A) :
    print("Yes");
else :
    print("No");
